src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -398,6 +402,13 @@ public IInternalContest fromYaml(IInternalContest contest, String[] yamlLines, S
             setCcsTestMode(contest, ccsTestMode);
         }
         
+        boolean stopOnFirstFail = fetchBooleanValue(content, STOP_ON_FIRST_FAILED_TEST_CASE_KEY, false);
+        if (stopOnFirstFail) {
```

Comment:
This "if" test doesn't make sense to me.  It says "if the 'stopOnFirstFail' boolean is TRUE, then update the setting".  It seems to me the setting should be updated whether it is true or false...

Reply:
removed if, pushed.